docs/examples/layouts/layoutsExample.js

```diff
@@ -0,0 +1,91 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Button, Flex, Heading, SelectList, TextField } from 'gestalt';
+
+export default function LayoutsExample(): Node {
+  return (
+    <Box padding={4}>
+      <Flex
+        direction="column"
+        gap={{
+          row: 0,
+          column: 6,
+        }}
+        maxWidth={800}
+        width="100%"
+        wrap
+      >
+        <Heading size="400" accessibilityLevel={2}>
+          Form Title
+        </Heading>
+
+        <TextField
+          label="TextField 1"
+          id="textfield1"
+          onChange={() => {}}
+          placeholder="Placeholder"
+        />
+
+        <Box
+          // Using Box instead of Flex + 'gap' for proper vertical spacing when text fields wrap
```

Comment:
now we no longer need this 😄

Reply:
@dangerismycat nice one, updated the example to use the general `gap` now.